src/Instance/InstanceDetails/InstanceDetails.js

```diff
@@ -57,24 +66,72 @@ const accordions = {
 const InstanceDetails = forwardRef(({
   children,
   instance,
-  paneTitle,
-  paneSubtitle,
   onClose,
   actionMenu,
   tagsEnabled,
+  isShared,
+  isLoading,
   ...rest
 }, ref) => {
-  const stripes = useStripes();
   const intl = useIntl();
+  const stripes = useStripes();
   const location = useLocation();
   const searchParams = new URLSearchParams(location.search);
 
   const referenceData = useContext(DataContext);
   const accordionState = useMemo(() => getAccordionState(instance, accordions), [instance]);
   const [helperApp, setHelperApp] = useState();
+
+  if (isLoading) {
+    return (
+      <Pane
+        id="pane-instancedetails"
+        defaultWidth="fill"
+        paneTitle={intl.formatMessage({ id: 'ui-inventory.edit' })}
+        appIcon={<AppIcon app="inventory" iconKey="instance" />}
+        dismissible
+        onClose={onClose}
+      >
+        <div style={{ paddingTop: '1rem' }}>
+          <Icon
+            icon="spinner-ellipsis"
+            width="100px"
+          />
+        </div>
+      </Pane>
+    );
+  }
+
   const tags = instance?.tags?.tagList;
   const isUserInCentralTenant = checkIfUserInCentralTenant(stripes);
 
+  const renderPaneTitle = () => {
+    const isInstanceShared = Boolean(isShared || isInstanceShadowCopy(instance?.source));
+
+    return (
+      <FormattedMessage
+        id={`ui-inventory.${isUserInConsortiumMode(stripes) ? 'consortia.' : ''}instanceRecordTitle`}
+        values={{
+          isShared: isInstanceShared,
+          title: instance?.title,
+          publisherAndDate: getPublishingInfo(instance),
+        }}
+      />
+    );
+  };
+
+  const renderPaneSubtitle = () => {
+    return (
+      <FormattedMessage
+        id="ui-inventory.instanceRecordSubtitle"
+        values={{
+          hrid: instance?.hrid,
+          updatedDate: getDate(instance?.metadata?.updatedDate),
+        }}
+      />
+    );
+  };
+
   const detailsLastMenu = useMemo(() => {
```

Comment:
Please fix eslint warning `React Hook "useMemo" is called conditionally.`

Reply:
done